### PR TITLE
Add --purge-persistent-cache hint to shaders.md

### PR DESCRIPTION
**Pull request by liyuqian**
This follows https://github.com/flutter/flutter/pull/67697 which
fixes https://github.com/flutter/flutter/issues/66473.

### CODE REVIEW

**Review by sfshaza2 — COMMENTED** [review]
Oops! This has been sitting here for ages. Finishing review.

File: src/docs/perf/rendering/shader.md [hunk 1]
```diff
@@ -60,6 +60,17 @@ and package the SkSL shaders:
 ```sh
 flutter run --profile --cache-sksl
 ```
+
+If the same app has been previously run without `--cache-sksl`, then
+`--purge-persistent-cache` flag may be needed:
```
[the remark this answers is not in the document]
Comment (liyuqian) [reply src/docs/perf/rendering/shader.md:65]: Done.

File: src/docs/perf/rendering/shader.md [hunk 2]
```diff
@@ -60,6 +60,17 @@ and package the SkSL shaders:
...
+
+```sh
+flutter run --profile --cache-sksl --purge-persistent-cache
+```
+
+Such flag removes older non-SkSL shader caches which could interfere with SkSL
```
[the remark this answers is not in the document]
Comment (liyuqian) [reply src/docs/perf/rendering/shader.md]: Done.

File: src/docs/perf/rendering/shader.md [hunk 3]
```diff
@@ -60,6 +60,17 @@ and package the SkSL shaders:
...
+shader capturing. It will also purge the SkSL shaders so one should only use it
```
[the remark this answers is not in the document]
Comment (liyuqian) [reply src/docs/perf/rendering/shader.md]: Done.

**Review by sfshaza2 — APPROVED** [review]
LGTM!

**Pull request merged**